docker/mediamtx-entry.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+[ -v DEBUG ] && set -x
+
+set -euo pipefail
+
+if [ -v LP_PUBLISH_MEDIAMTX_METRICS ]; then
+  if [ -z "$LP_PUBLISH_MEDIAMTX_METRICS_ENDPOINT" ]; then
+    echo >&2 "No endpoint specified for publishing mediamtx metrics."
+  fi
+  echo <<EOF |
+# HELP version Current software version as a tag, always 1
+# TYPE version gauge
+version{app="MediaMTX",version="$MEDIAMTX_VERSION"} 1
+EOF
+    curl -X POST --data-binary @- "$LP_PUBLISH_MEDIAMTX_METRICS_ENDPOINT"
+fi
+
+/usr/local/bin/mediamtx
```

Comment:
It could be:
```bash
exec ${1:-/usr/local/bin/mediamtx}
```

so it would work with `ENTRYPOINT` and `CMD` correctly. And `exec` to get rid of this intermediate bash script running forever.

Reply:
changed it to `${@:-...` so that complex commands would also work